test_schemas/en/test_view_submitted_response.json

```diff
@@ -0,0 +1,66 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Test View Submitted Response",
+    "theme": "default",
+    "description": "A questionnaire to test the use of view submitted response",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
+    "submission": {
+        "view_response": true
+    },
+    "sections": [
+        {
+            "id": "name-section",
+            "groups": [
```

Comment:
I think at one more group should be added to demonstrate that multiple groups render as expected. Multiple sections will be tested when hub support is added. But I don't think a single question is enough.

Reply:
We have tests for summary that are using the same summary context, so I don't think we need to have more than one, just that it works